EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -194,7 +194,7 @@ QUEST_LV_0300_20150908_000193	Looking at the device
 QUEST_LV_0300_20150908_000194	You feel like your spirit is moving out from your body as you touch the device!{nl}You should look for someone who can help you!
 QUEST_LV_0300_20150908_000195	The curse has been released
 QUEST_LV_0300_20150908_000196	Avoid the attacks of the monsters and destroy the terminal!
-QUEST_LV_0300_20150908_000197	Lighting up the fire after putting the bomb powder in place
+QUEST_LV_0300_20150908_000197	Place and ignite the explosive powder
```

Comment:
The original sentence implies you are doing it, so 'placing' would fit in here since you're doing the action.